src/core/Akka.Cluster.Tests/ClusterSpec.cs

```diff
@@ -20,6 +20,12 @@ namespace Akka.Cluster.Tests
 {
     public class ClusterSpec : AkkaSpec
     {
+
+        /*
```

Comment:
Added portability note explaining some of the semantic differences with the specs.